demos/demo_operation_log.sh

```diff
@@ -27,10 +27,10 @@ comment "The repo now looks like this:"
 run_command "jj log"
 comment "The most recent portion of the operation log
 is:"
-run_command "jj op log --color=always | head"
+run_command_allow_broken_pipe "jj op log | head"
 
 comment "Let's undo that rebase operation:"
-rebase_op=$(jj --color=never op log | grep '^◉  ' | sed '2q;d' | cut -b4-15)
+rebase_op=$(jj debug operation --display id --at-op @-- | head --bytes 8)
```

Comment:
With #2243 merged, you can possibly use `jj op log --no-graph -T id--at-op @--` and not need the `head`

Reply:
I changed it. I'm not sure this is easier to read, but it does avoid even a possibility of a broken pipe.